bundles/org.openhab.binding.radiothermostat/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,238 @@
+
+	<!-- RadioThemostat Thing -->
+	<thing-type id="rtherm">
+		<label>Thermostat</label>
+		<description>
+			A Thermostat to Control the House's HVAC System
+		</description>
+
+		<channels>
+			<channel id="name" typeId="name" />
+			<channel id="model" typeId="model" />
+			<channel id="temperature" typeId="temp-temperature" />
+			<channel id="humidity" typeId="humidity" />
+			<channel id="mode" typeId="mode" />
+			<channel id="fan_mode" typeId="fan_mode" />
+			<channel id="program_mode" typeId="program_mode" />
+			<channel id="set_point" typeId="temp-sp" />
+			<channel id="override" typeId="override" />
+			<channel id="hold" typeId="hold" />
+			<channel id="status" typeId="status" />
+			<channel id="fan_status" typeId="fan_status" />
+			<channel id="day" typeId="t_day" />
+			<channel id="hour" typeId="t_hour" />
+			<channel id="minute" typeId="t_minute" />
+			<channel id="dt_stamp" typeId="dt_stamp" />
+			<channel id="last_update" typeId="last_update" />
+			<channel id="today_heat_runtime" typeId="today_heat_runtime" />
+			<channel id="today_cool_runtime" typeId="today_cool_runtime" />
+			<channel id="yesterday_heat_runtime" typeId="yesterday_heat_runtime" />
+			<channel id="yesterday_cool_runtime" typeId="yesterday_cool_runtime" />
+			<channel id="json_cmd" typeId="json_cmd" />
+		</channels>
+
+		<config-description>
+			<parameter name="hostName" type="text" required="true">
+				<context>network-address</context>
+				<label>Thermostat Host Name/IP Address</label>
+				<description>Host Name or IP Address of the Thermostat</description>
+			</parameter>
+			<parameter name="refresh" type="integer" min="1" required="false">
+				<label>Refresh Interval</label>
+				<description>Specifies the Refresh Interval in Minutes</description>
+				<default>2</default>
+			</parameter>
+			<parameter name="logRefresh" type="integer" min="5" required="false">
+				<label>Run-time Log Refresh Interval</label>
+				<description>Specifies the Run-time Log and Humidity Refresh Interval in Minutes</description>
+				<default>10</default>
+			</parameter>
+			<parameter name="disableLogs" type="integer" min="0" max="1" required="false">
+				<label>Disable Retrieval of Run-time Data</label>
+				<description>Optional Flag to Disable the Retrieval of Run-time Data from the Thermostat</description>
+				<default>0</default>
+			</parameter>
+			<parameter name="disableHumidity" type="integer" min="0" max="1" required="false">
+				<label>Disable Retrieval of Humidity Data</label>
+				<description>Optional Flag to Disable the Retrieval of Humidity Data from the Thermostat</description>
+				<default>0</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="name" advanced="true">
+		<item-type>String</item-type>
+		<label>Thermostat Name</label>
+		<description>The Name of the Thermostat</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="model" advanced="true">
```

Comment:
This should be a thing property instead of a channel.

Reply:
removed